addons/survey/models/survey_survey.py

```diff
@@ -43,6 +43,13 @@ def default_get(self, fields_list):
         ('custom', 'Custom'),
     ],
         string='Survey Type', required=True, default='custom')
+    lang_ids = fields.Many2many(
+        'res.lang', string='Languages',
+        default=lambda self: self.env['res.lang']._lang_get(
+            self.env.context.get('lang') or self.env['res.lang'].get_installed()[0][0]),
```

Comment:
Checking if we should do something to have `False` at upgrade. 

@filisbits :)

Reply:
Nice catch also, thanks.